smart-integration/src/test/java/org/smartdata/integration/IntegrationTest.java

```diff
@@ -55,6 +53,7 @@ public static void setup() throws Exception {
     conf = cluster.getConf();
     httpHost = "localhost";
```

Comment:
Better to use "0.0.0.0" instead of using "localhost" as it may leads to test failures.